### ISSUE TITLE
WoW64 Support

### ISSUE THREAD

**Issue by Pobega** [flathub/com.fightcade.Fightcade#27]
https://github.com/fastrizwaan/flatpak-wine/blob/7ccd4bc71622361fd87af7b4c1275132c7e8fa4d/org.winehq.flatpak-wine/org.winehq.flatpak-wine.yml

**Comment by Pobega** [comment]
In case this gets lost, here is from the yml;

```yml
# build wow64              
  - name: org.winehq.wine_64bit
    builddir: true
    build-options:
      make-args: -j5
      prefix: /app
      prepend-path: /app/bin
      ldflags: -L/app/lib64 -L/app/lib32
      append-path: /usr/lib/sdk/toolchain-i386/bin
      env:
        CC: i686-unknown-linux-gnu-gcc
        CXX: i686-unknown-linux-gnu-g++
      libdir: /app/lib32
      strip: true
      no-debuginfo: true
      
    make-install-args:
      - LDCONFIG=/bin/true
      - STRIP=/bin/true
      - UPDATE_DESKTOP_DATABASE=/bin/true
    buildsystem: simple
    build-commands:
    # 1. Build wine64
    - export CC=x86_64-unknown-linux-gnu-gcc  &&
      export CXX=x86_64-unknown-linux-gnu-g++ &&
      mkdir wine64-build && cd wine64-build   &&
      ../configure 
      --prefix="/app" 
      --disable-win16
      --enable-shared      
      --disable-static
      --disable-tests          
      --with-x
      --without-cups
      --without-curses
      --without-capi
      --without-glu
      --without-gphoto
      --without-gsm
      --without-hal
      --with-ldap
      --with-faudio
      --without-netapi
      --enable-win64 &&  
      make -j5

    # 2. Build wine32 twice
    - export CC=i686-unknown-linux-gnu-gcc  &&
      export CXX=i686-unknown-linux-gnu-g++ &&
      mkdir wine32-tools && cd wine32-tools &&
      ../configure 
      --prefix="/app" 
      --disable-win16
      --enable-shared
      --disable-static
      --disable-tests          
      --with-x
      --without-cups
      --without-curses
      --without-capi
      --without-glu
      --without-gphoto
      --without-gsm
      --without-hal
      --with-ldap
      --with-faudio
      --without-netapi &&
      make -j5

    # 3. Build 32bit second time and 
    # install 32 bit and 64 bit wine
    - export CC=i686-unknown-linux-gnu-gcc  &&
      export CXX=i686-unknown-linux-gnu-g++ &&
      mkdir wine32-combo && cd wine32-combo &&
      ../configure 
      --with-wine64=../wine64-build
      --with-wine-tools=../wine32-tools
      --prefix="/app" 
      --disable-win16
      --enable-shared      
      --disable-static
      --disable-tests          
      --with-x
      --without-cups
      --without-curses
      --without-capi
      --without-glu
      --without-gphoto
      --without-gsm
      --without-hal
      --with-ldap
      --with-faudio
      --without-netapi   &&
      make -j5           && 
      make install       && 
      cd ../wine64-build && 
      make install

    sources:
      - type: archive
        url: https://dl.winehq.org/wine/source/5.0/wine-5.0.4.tar.xz
        sha512: 6ece6e2eda95f960c876577c9303ea53703975db60af2150ed6583082a2ad700a0bcd92d51190843fcb39d701b215e035376ff401fe822db5bee9650cb7f4d00
```

**Comment by Pobega** [comment]
Resolved by #28
